generators/server/cleanup-reactive.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
If we split reactive, we may need to split every other tech into imperative/reactive.
We probably should keep it in server.

Reply:
I suggest to have a `clean-reactive` file for everything that is common, but use `if(reactive)`in the respective `cleanup-<tech>` for everything related to a particular technology (sql, angular, ...)